guides/guide-for-adding-windows-node.md

```diff
@@ -159,21 +159,30 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-t
 
 > **Note:** All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Administrator) on the Windows worker node.
 
-1. Install ContainerD, wins, kubelet, and kubeadm.
+1. Install ContainerD.
 
 ```PowerShell
 # Install ContainerD
 curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/Install-Containerd.ps1
-.\Install-Containerd.ps1
+.\Install-Containerd.ps1 -ContainerDVersion 1.7.1 -netAdapterName Ethernet
```

Comment:
```suggestion
.\Install-Containerd.ps1 -ContainerDVersion 1.7.1 
```
This is still the default even with the changes.

Reply:
That's true. I just thought it's more clear for new users which values will be used if we specify them explicitly.
Note that I also changed to default value for `ContainerDVersion` to `1.7.1`.